test/TensorFlow/diagnostics.swift

```diff
@@ -102,19 +102,19 @@ public func inoutArgumentToAccelerator(t: inout Tensor<Float>) {
   t += 1  // expected-note {{value used here}}
 }
 
-// Opaque handles should not be sent or received.
-public func opaqueHandlesCannotBeSentOrReceived() {
-  // expected-error @+2 {{value cannot be used by non-TensorFlow API}}
+// Resource handles may be sent or received.
+public func resourceHandlesCanBeSentOrReceived() {
```

Comment:
can you confirm from SIL code that `iterator` involves any sends/recvs? I believe it is used as an input arg to the tensor program. If so, pls rename the test case name.